doc/man3/OSSL_CMP_SRV_CTX_new.pod

```diff
@@ -21,7 +21,8 @@ OSSL_CMP_SRV_CTX_get0_custom_ctx,
 OSSL_CMP_SRV_CTX_set_send_unprotected_errors,
 OSSL_CMP_SRV_CTX_set_accept_unprotected,
 OSSL_CMP_SRV_CTX_set_accept_raverified,
-OSSL_CMP_SRV_CTX_set_grant_implicit_confirm
+OSSL_CMP_SRV_CTX_set_grant_implicit_confirm,
+OSSL_CMP_SRV_CTX_centralKeygen_req
```

Comment:
This function MUST be added explicitly to the RETURN VALUES section due to its special behavior.